docs/content/config.md

```diff
@@ -91,15 +91,17 @@ Workspacer offers additional functionality beyond just tiling your windows throu
 
 ### Menu Bar
 
-Like other tiling window managers workspacer includes a status/menu bar that show relevant information to improve your workflow, for example: showing your workspaces, your battery level and the time. It can also have additional features provided through custom widgets by creating a class which inherits from [BarWidgetBase](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Bar/BarWidgetBase.cs). <br/> The bar can be installed like this:
+Like other tiling window managers workspacer includes a status/menu bar that show relevant information to improve your workflow, for example: showing your workspaces, your battery level and the time. It can also have additional features provided through custom widgets by creating a class which inherits from [BarWidgetBase](https://github.com/workspacer/workspacer/blob/master/src/workspacer.Bar/BarWidgetBase.cs).
```

Comment:
I think "...that show..."  should be "...that shows..." here right?

Reply:
Fixed [here](https://github.com/workspacer/workspacer/pull/229/commits/e4aa7b22dc740f40a00515dd3f56fcae9a2c5eed)

Reply:
Yeah I think you're right. Though I think it's technically "which shows", but that's really nitty gritty.

Reply:
True! On it

Reply:
[Resolved](https://github.com/workspacer/workspacer/pull/229/commits/f200e896da297d7a94f66ed5569eb7d0b082bdfa)